lib/fog/google/models/storage_json/directories.rb

```diff
@@ -0,0 +1,37 @@
+require "fog/core/collection"
+require "fog/google/models/storage_json/directory"
+
+module Fog
+  module Storage
+    class GoogleJSON
+      class Directories < Fog::Collection
+        model Fog::Storage::GoogleJSON::Directory
+
+        def all
+          data = service.get_service.body["items"]
+          load(data)
+        end
+
+        def get(key, options = {})
+          remap_attributes(options,             :delimiter  => "delimiter",
+                                                :marker     => "marker",
+                                                :max_keys   => "max-keys",
+                                                :prefix     => "prefix")
+          data = service.get_bucket(key, options).body
+          directory = new(:key => data["name"])
+          # options = {}
```

Comment:
Dunno. If is not used or does not make sense we can rip it out but i'm not that into the platform to say. :sweat: